gettsim/taxes/zve.py

```diff
@@ -111,7 +111,11 @@ def kinderfreibetrag(tax_unit, params, kindergeld_params):
     nokfb_lower = tax_unit["zve_nokfb"].min()
 
     # Add both components for ease of notation.
-    kifreib_total = params["kifreib_s_exm"] + params["kifreib_bea"]
+    try:
+        kifreib_total = params["kifreib_s_exm"] + params["kifreib_bea"]
+        # 'kifreib_bea' does not exist before 2000.
+    except NameError:
+        kifreib_total = params["kifreib_s_exm"]
```

Comment:
I don't think this is good way to do these things:
* If you add a clear year condition in the source code it would be enough to look at the source code (no need to investigate the param files further).
* This type of check does not allow to model cases where a law becomes obsolete.
* This type of check does not allow to model cases where a law does not exist for a few years and has been introduced later (e.g. Soli only in 1991, 1992 and then again 1995).

Reply:
I think it's a way to deal with this particular issue. The alternative would be to introduce separate functions in `policy_for _date`, which again is stupid because there is no structural change in the law whatsoever. For larger changes like soli, one has to go the long way.

Reply:
This looks more like a quick hack to me. I would say it is a structural change.

What is wrong with adding it to `policy_for_date`? Is the problem that the `policy_for_date` way does not scale? (we will have a lot of this type of policy changes). What is wrong with a year switch? That would be easy to read and robust to the two other problems I mentioned.

Reply:
> What is wrong with adding it to `policy_for_date`? Is the problem that the `policy_for_date` way does not scale? (we will have a lot of this type of policy changes). 

Precisely. I can do a year switch although we wanted to avoid them.

Reply:
In the long run, yes. For now, they are fine.